pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerStatsDisabled.java

```diff
@@ -145,7 +146,12 @@ public void reset() {
     }
 
     @Override
-    public void updateCumulativeStats(ConsumerStats stats) {
+    public void updateCumulativeStats(String partition, ConsumerStats stats) {
```

Comment:
1. If we change the method singuture, I'm not sure will it bring some compatible problem, because this method is public.

2. The name of this method is  `updateCumulativeStats`,  doing some partitioned stats work is not `cumulative` I think.  How about add a new method like `updatePartitionedStats()` :)

Reply:
this method is only used by partitioned topic stats only and it's not part of public interface and used by internal Impl class only with single thread. so, there is no issue in changing the signature.

Reply:
I have removed the api change.